### DISCUSSION TITLE
Codespaces memory usage max'd out.

### DISCUSSION THREAD

**Discussion by painotpi** [community/community#40267] [Codespaces] [answered]
### Select Topic Area

Question

### Body

GitHub sent me alerts over the last week once I crossed the memory usage thresholds (75%, 90%, and 100%).
However, I don't recall enabling codespaces, nor have I used it in the past 2 months. How/Why am I breaching the memory limits?

**Comment by GideonCheruiyot** [comment, marked as answer]
Hi @painotpi, we are aware of the issue and are working on handling the notifications. Here's a post on the some [updates](https://github.com/orgs/community/discussions/39697).
